From a28479f374b5aba27a794a55578cd61d87a5e949 Mon Sep 17 00:00:00 2001
Message-ID: <a28479f374b5aba27a794a55578cd61d87a5e949.1713654571.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1713654571.git.sharlatanus@gmail.com>
References: <cover.1713654571.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Tue, 2 Apr 2024 21:15:16 +0100
Subject: [PATCH 17/47] gnu: python-stcal: Update to 1.7.0.

* gnu/packages/astronomy.scm (python-stcal): Update to 1.7.0.
[native-inputs]: Remove python-pytest-openfiles.

Change-Id: Ib33bb72ff73fcf569a2519b3353260461b63c143
---
 gnu/packages/astronomy.scm | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/gnu/packages/astronomy.scm b/gnu/packages/astronomy.scm
index 374229bcf4..cb5be9393d 100644
--- a/gnu/packages/astronomy.scm
+++ b/gnu/packages/astronomy.scm
@@ -3229,13 +3229,13 @@ (define-public python-stsci-stimage
 (define-public python-stcal
   (package
     (name "python-stcal")
-    (version "1.6.1")
+    (version "1.7.0")
     (source
      (origin
        (method url-fetch)
        (uri (pypi-uri "stcal" version))
        (sha256
-        (base32 "1rwvzjpidz0sfp52mqgs25599hbvxyc4gdc1yk0kdm8aynhdwi82"))))
+        (base32 "04ippj90dn2671xg4rvncf7jsbpcflr7dr6jc2h52z7xfd1msb4d"))))
     (build-system pyproject-build-system)
     (arguments
      (list
@@ -3264,7 +3264,6 @@ (define-public python-stcal
            python-pytest
            python-pytest-cov
            python-pytest-doctestplus
-           python-pytest-openfiles
            python-semantic-version
            python-setuptools-scm))
     (home-page "https://github.com/spacetelescope/stcal")
-- 
2.41.0

